Sources/DistributedActors/Timers+Distributed.swift

```diff
@@ -149,6 +147,9 @@ public final class ActorTimers<Act: DistributedActor> where Act.ActorSystem == C
                 }
 
                 await call()
+
+                // The single timer is done. Remove it so it can be installed again if needed.
+                self.cancel(for: key)
```

Comment:
This is called from another thread so we need to lock around all updates of     `internal var installedTimers: [TimerKey: DistributedActorTimer] = [:]` now;

or we could turn this into an actor but since this type needs to go away anyway, let's just do the lock to keep it simple.

Reply:
amended https://github.com/apple/swift-distributed-actors/pull/966/commits/f326d38e60c0540339faa3259ab60eb5bcbfa580